zdict/tests/dictionaries/test_spanish.py

```diff
@@ -0,0 +1,42 @@
+from pytest import raises
+from unittest.mock import Mock, patch
+
+from zdict.dictionaries.spanish import SpanishDict
+from zdict.exceptions import NotFoundError
+
+
+class TestSpansishDict:
+    d = SpanishDict()
+    word = 'Spanish'
+    timeout = 5
+    raw_html = d._get_raw(word, timeout)
+    record = d.query(word, timeout)
```

Comment:
I will suggest that pust line 9-13 in a setup/teardown method.
ref: https://pytest.org/latest/xunit_setup.html#method-and-function-level-setup-teardown
e.g.
```python
class TestSpansisDict:
  def setup_method(self, method):
     self.d = SpanihDict()
      ...
  def teardown_method(self, method):
     del self.d
     del ...
```
This snippet can ensure our each testing method not to be effect by other else testing method. (or say isolation our testing method).
I think those vars (`self.d`, `self.record` ... etc) can consider as kind of fixture, so prepare them in `setup` methd will be resonable.